.github/workflows/artifacts.yml

```diff
@@ -84,6 +84,10 @@ jobs:
         if: github.event_name == 'release'
         run: npx nx run-many --projects=${{ env.APP_NAMES }} --target=docker-build
 
+      - name: Build all docker images as latest
+        if: github.event.pull_request.merged == true
```

Comment:
I'm not sure this will trigger (this workflow does not react on pull request events); I think a better condition could be:
```
if: github.ref_name != 'main'
```